operator/v1/types_console.go

```diff
@@ -146,6 +146,14 @@ type ConsoleCustomization struct {
 	// +listMapKey=id
 	// +optional
 	Perspectives []Perspective `json:"perspectives"`
+	// consoleURL is an URL to be used as the base console address.
+	// If not specified, the console route hostname will be used.
+	// This is required when using console on clusters where the ingress capability is disabled.
```

Comment:
What happens when you try to use this value when ingress is not disabled?

Reply:
That part should be programatically conditioned by the console-operator

Reply:
OK, but what does the operator do? Is the value ignored?

Reply:
The only use case for the moment is when the ingress is disabled. The comment needs to be updated to highlight this. I think the implementation may be expanded in the future as use cases come up.

Reply:
Field comment updated.